### PR TITLE
Use same spacing for class body, return types. Fix phpdocs

### PR DESCRIPTION
**Pull request by andrew-demb**

### CODE REVIEW

**Comment by funivan** [comment]
Thanks for the PR/

This part of README is autogenerated. You need to edit HTML files and then run `./gradlew updateReadme` 

For example:
https://github.com/funivan/PhpClean/blob/009926d9bed97ab051b1d49da4e5526c6ef30c11/src/main/kotlin/com/funivan/idea/phpClean/inspections/assignMisused/AssignMisusedInspection.html

File: README.md [hunk 1]
```diff
@@ -21,20 +21,20 @@ Hit `install` button.
 #### AssignMisused
 Detects assignment and comparison operators in one statement.
 ```php
-do{
+do {
  //...
-}while( false == $user->getName() = self::ADMIN_USER );
+} while( false == $user->getName() = self::ADMIN_USER );
```
Comment (andrew-demb) [inline README.md:26]: This statement cannot be executed in PHP.
Cannot find a proper alternative example, but it should be updated once.

Comment (funivan) [reply README.md:26]: I think `while (false == $admin = $user->getName() )` should be good here, WDYT?

**Review by funivan — APPROVED** [review]
Thanks. 👍

File: README.md (hunk 1 above)
Comment (andrew-demb) [reply README.md:26]: Falsable username is not clear to the developer for an understanding of the issue.
Also, I cannot find a case for assigning in do-while to reuse it later. 

An adapted current example may be like this.
```php
$adminReceived = false;
do {
  // false
while (false === $adminReceived = $user->isAdmin());
```

Comment (funivan) [reply README.md:26]: In most cases, variables are changed in a `do` body.
Your example is good. 

BTW example from the Symfony framework:
```php
while (false !== $current = ldap_next_entry($con, $current)) {
  yield $this->getSingleEntry($con, $current);
}
```

**Pull request merged**